src/TabViewPagerAndroid.js

```diff
@@ -94,13 +94,14 @@ export default class TabViewPagerAndroid<T: Route<*>> extends PureComponent<
 
   _setPage = (index: number) => {
     if (this._viewPager) {
+      // in spite of this check, flow 0.53.0 still reports errors if `this._viewPager &&` is omitted below
```

Comment:
Flow doesn't allow you to refine properties on `this`. To get around this, assign to a locally scoped variable, and  refine the type of the locally scoped variable.

```js
const viewPager = this._viewPager;
if (viewPager) {
```